Makefile

```diff
@@ -0,0 +1,55 @@
+# -------------------------------------------------------------
+# This makefile defines the following targets
+#   - tape - builds a binary program
+#   - tape-docker - build the tape image
+#   - unit-test - runs the go-test based unit tests
+#   - integration-test - runs the integration tests
+
+FABRIC_VERSION = latest
+INTERGATION_CASE = ANDLogic
+
+BASE_VERSION = 0.0.2 

```

Comment:
so we need upgrade version in make file pre lease?
or will we use some git branch name? (get from CI?)